Scripts/Runtime/Entities/Lifecycle/EntityLifecycleStatus.cs

```diff
@@ -243,9 +244,9 @@ private struct UpdateArrivedJob : IJobEntityBatch
                 m_ArrivedEntities = arrivedEntities;
             }
 
-            public void Execute(ArchetypeChunk batchInChunk, int batchIndex)
+            public void Execute(in ArchetypeChunk chunk, int unfilteredChunkIndex, bool useEnabledMask, in v128 chunkEnabledMask)
```

Comment:
No because this is concerned with knowing when Entities exist in the world. Their status of being enabled or not is something that would be checked further downstream by something that cares about it. 

If we checked the flag, we would always miss an entity that entered the world disabled.

Reply:
Shouldn't we configure that in the query that this job operates on instead though?

Fair enough if skipping the check is an optimization. We should just have a comment to explain why we're skipping it and mentioning that our query is also requesting all disabled entities.

There's also a risk here that today, or in the future, there is a chunk level optimization where Unity will skip a chunk if all entities in that chunk are disabled and the query isn't requesting to find disabled entities as well.

Reply:
Good point, in reading deep into the docs, if an entire chunk is disabled, it will skip the chunk. 

The query for this will now ensure it includes disabled entities and disabled components.

Reply:
Can you make sure to add the comment about why you're ignoring the mask?

Reply:
Yep, done!